docs/user/masternodes/hosting.rst

```diff
@@ -126,7 +126,8 @@ https://nodehub.io
 
 - Operated by: NodeHub LLC
 - Services: Hosting, Governance, Stats, Monitoring
-- Cost: $3.90/month (charged daily at $0.13)
+- Cost: $4.8/month (charged daily at $0.16)
```

Comment:
This page still lists $3.90: https://nodehub.io/pricing. Where is the updated value coming from?
```suggestion
- Cost: $4.80/month (charged daily at $0.16)
```

Reply:
We've recently updated our pricing for Dash from $3.90 to $4.80. New pricing can be seen on the deployment page; https://nodehub.io/dashboard/deploy?coin=dash
<img width="434" alt="image" src="https://github.com/user-attachments/assets/1c290ba2-1dee-48d8-8cdf-839efdfb7912">
Pricing page will be updated at a later date.